CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.16)
 if (POLICY CMP0048)
```

Comment:
Can you get rid of this if (and keep in policy set) while bumping the CMake version?

It is no longer necessary, CMP0048 is present in CMake >= 3.0.

Reply:
Same in the other CMakeLists.txt files:
- [src/gui/shellwidget/CMakeLists.txt](https://github.com/equalsraf/neovim-qt/blob/edd4b9f5d570343aee8016b54abd1f82754dccee/src/gui/shellwidget/CMakeLists.txt)
- [third-party/CMakeLists.txt](https://github.com/equalsraf/neovim-qt/blob/9dbd99dfaccd52a8b24b2c914831e5c7b9de6e28/third-party/CMakeLists.txt)
- [CMakeLists.txt](https://github.com/equalsraf/neovim-qt/blob/409fad81323f44e83ebcdbd4e8fa13bfff3f844c/CMakeLists.txt)